packages/liferay-npm-scripts/src/scripts/lint.js

```diff
@@ -85,25 +101,34 @@ function lint(options = {}) {
 		report = {results: []};
 	}
 
-	jspPaths.forEach(filePath => {
-		// TODO: non-sync version to make use of I/O concurrency
-		const contents = fs.readFileSync(filePath, 'utf8');
-
-		const updated = lintJSP(
-			contents,
-			result => {
-				report.results.push({
-					...result,
-					filePath: path.resolve(filePath)
-				});
-			},
-			{fix, quiet}
-		);
-
-		if (fix && updated !== contents) {
-			fs.writeFileSync(filePath, updated);
+	for (const [paths, linter] of [
+		[jspPaths, lintJSP],
+		[scssPaths, lintSCSS]
+	]) {
+		for (const filePath of paths) {
+			// TODO: non-sync version to make use of I/O concurrency
+			const contents = fs.readFileSync(filePath, 'utf8');
+
+			try {
+				const updated = await linter(
+					contents,
+					result => {
+						report.results.push({
+							...result,
+							filePath: path.resolve(filePath)
+						});
+					},
+					{filePath, fix, quiet}
+				);
+
+				if (fix && updated !== contents) {
+					fs.writeFileSync(filePath, updated);
+				}
+			} catch (error) {
+				log(error);
```

Comment:
I don't expect to see errors here (old code path didn't both `catch`-ing them, and I didn't see any in testing) but doing this out of an excess of caution seeing as I don't know how trustworthy the stylelint codebase is yet. At least in initial testing stylelint seems to degrade pretty gracefully; for example, a syntax error gets reported like any other lint error:

```
modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/css/main.scss
  98:6  error  Unexpected } (CssSyntaxError)  CssSyntaxError
```

Reply:
Interesting... is that a `CSS Syntax Error` or an `SCSS Syntax Error`? Are we compiling then parsing?

Reply:
I inserted a bunch of extra `}}}}` in an ".scss" file.

We're only linting ".scss" files, and we're passing `syntax: 'scss'` to stylelint, so it is doing the parsing.